lisa/tools/ip.py

```
@@ -215,6 +215,18 @@ def restart_device(
             ),
         )
 
+    def is_valid_mtu(self, value: Optional[int] = None) -> bool:
+        if value is None:
+            return False
+        try:
+            mtu = int(value)
+            # typical MTU values are between 576 and 9000
+            # 576 is the minimum possible MTU for IPv4 and 1280 for IPv6
+            # 9000 is a common value for jumbo frames on Ethernet networks
+            return 576 <= mtu <= 9000
```

If someone sets the value to 570 but it runs with a different value, it should raise an exception and fail the case instead of silently using the default.

---

set_mtu has this logic already.

---

Based on the logic, if the MTU is out of range, the function returns false and sets MTU to `None`, which is unexpected. If you want `set_mtu` to fail via command, remove the check here.

---

Fixed as suggested. Included the mtu check into set_mtu method.